libraries/resource_providers/api2200/c7000/logical_interconnect_provider.rb

```diff
@@ -0,0 +1,26 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API2200
+    module C7000
+      # LogicalInterconnect API2200 C7000 provider
+      class LogicalInterconnectProvider < API2000::C7000::LogicalInterconnectProvider
+        def validate_bulk_inconsistency
```

Comment:
this function is re-defined here, since its  already available in API 2000.

Reply:
yes, no need to add that method again in API 2200 since it is already added in API2000 library and inheriting API2000 libraries will inherit that method too. Please delete this.

Reply:
Done